packages/taquito/test/wallet/batch-wallet-operation.spec.ts

```diff
@@ -48,7 +48,7 @@ describe('BatchWalletOperation', () => {
 
       flush();
 
-      expectObservable(confirmation).toBe('--a', {
+      expectObservable(confirmation).toBe('----a', {
```

Comment:
The code change looks good to me, but I am curious about the change in the test timeframe. It does not appear to be an issue, though, as all integration tests still pass.

Reply:
It's because of `resetOnRefCountZero: true` which it'll torn down the observable when it's not subscribe. 
To resubscribe it'll start from scratch which caused the delay